egs/timit/asr/simple_v1/local1/timit.py

```diff
@@ -0,0 +1,147 @@
+  
```

Comment:
I suggest to contribute this recipe to Lhotse and import it from there.

Reply:
OK, I will do it.